vault/resource_quota_lease_count.go

```diff
@@ -143,6 +155,12 @@ func quotaLeaseCountUpdate(d *schema.ResourceData, meta interface{}) error {
 	data["path"] = d.Get(consts.FieldPath).(string)
 	data["max_leases"] = d.Get("max_leases").(int)
 
+	if data["path"] == "" && d.Get("inheritable") == nil {
+		data["inheritable"] = true
+	} else if v, ok := d.GetOkExists("inheritable"); ok {
+		data["inheritable"] = v.(bool)
+	}
```

Comment:
GetOkExists is deprecated so we prefer to use d.Get for booleans. This can be simplified and requires `inheritable` to not be Computed.

```suggestion
	data["inheritable"] = d.Get("inheritable").(bool)
```